src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -775,15 +766,15 @@ protected void HandleOutputDataReceived(string data)
             {
                 // Route protocol message based on whether it is a session or command message.
```

Comment:
Please move string null/empty check to here.

Reply:
I added the explicit check.